lib/model/CProbabilityAndInfluenceCalculator.cc

```diff
@@ -246,19 +280,24 @@ class CMeanDifference {
             TDouble2Vec{n[0] / (n[0] - ni[0]), n[1] / (n[1] - ni[1])},
             params.weights()[0]);
         params.addBucketEmpty(bucketEmpty);
+
+        return true;
     }
 };
 
 //! \brief Computes the value of the variance statistic on a set difference.
 class CVarianceDifference {
 public:
     //! Features.
-    void operator()(const TDouble1Vec& v,
```

Comment:
Maybe document the parameters to this function too? i.e. `v` == overall variance and mean, `n` == overall count, `vi` == influencer variance and mean, etc.